### DISCUSSION TITLE
Issue with an assertion method to match any error or panic behavior

### DISCUSSION THREAD

**Discussion by AkechiShiro** [Homebrew/discussions#3582] [Writing Formulae/Casks] [answered]
Hi, 

I'm trying to use `assert_no_match` or `assert_not_match` but none of these methods exists for formula testing.

Here is the error, I'm getting when I put inside the test block the `assert_no_match` method : 
```
NoMethodError: undefined method `assert_no_match' for #<Formulary::FormulaNamespace22ac6f203a9ee4cf22ddfd95258ceb1f::FormulaeName:0x00007fe834a85e70>
Did you mean?  assert_match
```

But it is being used in this formula just fine: https://github.com/Homebrew/homebrew-core/blob/6866149bd8f6ec8fe8323589227f0361264b027d/Formula/sslyze.rb

**Comment by carlocab** [comment, marked as answer]
`sslyze` hasn't been using `assert_no_match` for over a year now: https://github.com/Homebrew/homebrew-core/commit/23051733fee68d18c74d9c309b6963de04450196
